L1Trigger/L1THGCal/test/testHGCalL1T_RelVal_cfg.py

```diff
@@ -65,10 +65,12 @@
 # load HGCAL TPG simulation
 process.load('L1Trigger.L1THGCal.hgcalTriggerPrimitives_cff')
 process.hgcl1tpg_step = cms.Path(process.hgcalTriggerPrimitives)
+process.hgcalConcentratorProducer.ProcessorParameters.Method = cms.string('superTriggerCellSelect')
```

Comment:
The default values should be kept here. 
But for easy customization, it would be good to add a customization function, in the same way it is done for clustering:
https://github.com/PFCal-dev/cmssw/blob/hgc-tpg-devel-CMSSW_10_3_0_pre4/L1Trigger/L1THGCal/python/customClustering.py

Reply:
I have added a new file - let me know if it was what you were thinking of